src/Search/Search.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import Card from '../Card/Card'; // Update the path as per your file structure
+
+function WikipediaSearch() {
+  const [searchInput, setSearchInput] = useState('');
+  const [results, setResults] = useState([]);
+
+  const handleInputChange = (e) => {
+    setSearchInput(e.target.value);
+  };
+
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
+    if (searchInput.trim() !== '') {
+      displaySearchResults(searchInput);
+    }
+  };
+
+  const displaySearchResults = (searchTerm) => {
+    let url = `https://en.wikipedia.org/w/api.php?action=query&list=search&prop=info&inprop=url&utf8=&format=json&origin=*&srlimit=20&srsearch=${searchTerm}`;
```

Comment:
Ah okay so this is the API we are using. I will still test this endpoint etc and share with you all!